client/src/views/Dataset.vue

```diff
@@ -128,8 +109,9 @@ export default {
   watch: {
     currentSession(session) {
       if (session) {
-        this.rating = session.rating;
-        // TODO this.reviewer should also be set here
+        this.decision = session.decision;
+        this.decisionChanged = false;
+        this.newNote = null;
```

Comment:
Is there any reason this is null and not an empty string? In `save()`, calling `this.newNote.trim()`  will throw an error unless you modify the note (i.e. so it doesn't become null)

Reply:
Good catch, I missed that.